lib/bolt/outputter/human.rb

```diff
@@ -8,6 +8,15 @@ class Human < Bolt::Outputter
                  green: "32",
                  yellow: "33" }.freeze
 
+      BUILTIN_MODULES = %w[
```

Comment:
We've previously addressed this with https://github.com/puppetlabs/bolt/blob/master/lib/bolt/cli.rb#L416. We should make sure we don't have 2 methods.

Reply:
There are more built-in modules than are included in this list. I think we should use the previous method.

Reply:
It looks like that only includes the name of the task, not the path? I'm worried that people will make their own tasks with the same name, and those will accidentally be caught and labelled as builtin.

Reply:
Yeah, that'd be less helpful.

Reply:
Yeah we definitely support that. I think we shouldn't care about the task names at all and just call anything installed in the builtin moduledir a builtin